src/components/ToDoList/ToDoList.test.tsx

```diff
@@ -3,33 +3,42 @@ import { ToDoList } from "./ToDoList";
 import * as toDoService from "../../services/toDoService";
 import { faker } from "@faker-js/faker";
 import userEvent from "@testing-library/user-event";
+import { ToDoItem } from "../../services/toDoItem";
 
 describe("ToDoList", () => {
     // true unit test
-    it("should load list items", () => {
+    it("should load list items", async () => {
         // Arrange
-        const toDoItems = [faker.lorem.word(), faker.lorem.word()];
-        jest.spyOn(toDoService, "getToDoList").mockReturnValue(toDoItems);
+        const toDoItems = createToDoItems(2);
+        jest.spyOn(toDoService, "getToDoList").mockImplementation(() => Promise.resolve(toDoItems));
```

Comment:
```suggestion
        jest.spyOn(toDoService, "getToDoList").mockResolvedValue(toDoItems);
```

There's a little syntactic sugar available for when you need to mock a Promise's return value. Do you think it's worth using? Or do you prefer the way you have it, considering this is for educational purposes?

Reply:
I'll look this up

Reply:
Found it!